client/src/components/comment-form/comment-form.js

```diff
@@ -103,7 +102,7 @@ const CommentForm = ({
           key={comment._id}
           text={comment.text}
           date={comment.date}
-          reviewerName={comment.user.firstName}
+          reviewerName={comment.user.firstName || "Anonym"}
```

Comment:
remove hardcode "Anonym" to 
const anonim = "Anonim" than use 
reviewerName={comment.user.firstName || anonim}